ui/src/pages/ShoeMap.tsx

```diff
@@ -7,25 +7,22 @@ import { PinCluster } from "../components/PinCluster";
 import { StoryDrawer, StoryDrawerState } from "../components/StoryDrawer";
 import { useState } from "react";
 import { colors } from "../styles";
+import ShoeLogo from "../assets/images/shoeproject-logo.svg";
 
 const StyledMap = styled(Map)`
   height: 100vh;
   width: 100vw;
-
   .leaflet-control {
     border: none;
-    box-shadow: 0px 0px 25px rgba(0, 0, 0, 0.1);
```

Comment:
took this out because it's adding a redundant box shadow on all control components